verifiers/internal/gha/npm.go

```diff
@@ -144,6 +127,12 @@ func (n *Npm) verifyPublishAttestationSignature() error {
 		return err
 	}
 
+	// Retrieve the key material.
+	npmRegistryPublicKey, err := getKeyDataFromSigstoreTuf(npmRegistryPublicKeyID, attestationKeyUsage)
```

Comment:
iiuc, the inner tuf client will use a local cache, so it's really only fetched from the internet once per invocation of `getKeyDataFromSigstoreTuf`

Reply:
by "local", you mean in memory or on disk?

Reply:
On disk. But now I think about it, perhaps @ianlewis meant the repeated verifications that GetTarget will do are expensive. Even if it did not do a repeated verification, it may still be better to not rely on the local cache on disk. I'll change it to preserve the final keydata in slsa-verifier memory.

Reply:
I was really trying to make sure we don't go to the internet each time, but yeah, if we can avoid hitting the disk too so much the better.